README.md

```diff
@@ -32,32 +34,37 @@ git submodule update
 
 Set the following environment variables:
 
-| Variable name         | Definition                                                                                                                       |
-| --------------------- | -------------------------------------------------------------------------------------------------------------------------------- |
-| `VCPKG_DIR`           | The path to an extracted release of Trail of Bits' cxx-common repository. This provides a path to all of Macroni's dependencies. |
-| `LLVM_LIT`            | The path to your install of `llvm-lit`. This enables automated testing of Macroni.                                               |
-| `MACRONI_INSTALL_DIR` | The directory to install Macroni to.                                                                                             |
-| `MACRONI_SRC`         | The directory macroni has been downloaded to.                                                                                    |
-| `MACRONI_BUILD_DIR`   | The directory to build Macroni in.                                                                                               |
+| Variable name          | Definition                                                                         |
+| ---------------------- | ---------------------------------------------------------------------------------- |
+| `CMAKE_INSTALL_PREFIX` | The path to where you would like to install Macroni to.                            |
+| `LLVM_EXTERNAL_LIT`    | The path to your install of `llvm-lit`. This enables automated testing of Macroni. |
+| `CMAKE_TOOLCHAIN_FILE` | The path to your `cxx-common` version's toolchain file (`vcpkg.cmake`).            |
+| `CMAKE_C_COMPILER`     | The path to your `cxx-common` version's C compiler.                                |
+| `CMAKE_CXX_COMPILER`   | The path to your `cxx-common` version's C++ compiler.                              |
 
+**Tip**: If you are developing in Visual Studio or Visual Studio Code, use the
+provided CMakePresets.json file to make your development easier. Just edit the
```

Comment:
You can use them even if you build on command line? Or at least you should provide some for that use case as well - instead of that pretty long invocation down the line.